cpp/ps0001/ps0001solution.cpp

```diff
@@ -1,157 +1,25 @@
-﻿
 #include <iostream>
 #include "hadhod.cpp"
 #include "easylearn.cpp"
 using namespace std;
 
-
-
-
-
-
-
-
-
-
 int main()
 {
+    // prompting user for 3 numbers to add 1/2
     cout << "Enter the first number: \n";
-    float num1;
+    unsigned short num1;
     cin >> num1;
 
     cout << "Enter the second number: \n";
-    float num2;
+    unsigned short num2;
     cin >> num2;
 
     cout << "Enter the third number: \n";
-    float num3;
+    unsigned short num3;
     cin >> num3;
 
-    cout << num1 << endl << num2 << endl << num3 << endl << "\n---------------\n" << num1 + num2 + num3;
-    return 1;
+    // outputting the result for summation process 3/3
+    cout << "---------------\n" << num1 + num2 + num3 << endl;

```

Comment:
- [x] did summation process for the three inputted numbers
- [x] used `std::cout` in the right way to display output from the summation process
- [x] output is as required
```
155
940
168
--------
1263
```
+1 bonus mark as program is user-friendly